### ISSUE TITLE
Bundler error: bundler-1.8.5/lib/bundler/cli.rb:20:in `initialize': undefined method `[]' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by chourobin** [MaximeD/gem_updater#4]
Hey, awesome idea for this gem. Does this require an older version of bundler? Running gives the following error:

```
bundler-1.8.5/lib/bundler/cli.rb:20:in `initialize': undefined method `[]' for nil:NilClass (NoMethodError)
```
```
Bundler::CLI.new.update
```

**Comment by mattbrictson** [comment]
I'm also using bundler 1.8.5 and see the same error.

**Comment by chourobin** [comment]
Fixed for me by using

`
Bundler::CLI.start(['update'])
`

**Comment by mattbrictson** [comment]
@chourobin Can you reopen this issue? I think it should stay open until the problem is addressed (either by merging #5 or some other fix).

**Comment by MaximeD** [comment]
Support for bundler 1.8 has been merged. You can update `gem_updater` and enjoy it. Thanks for your feedback
